src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/article/editmode/SourceEditModePageObject.java

```diff
@@ -76,7 +75,7 @@
   private WebElement sourceModeTextArea;
 
   public SourceEditModePageObject openArticle(String articleTitle) {
-    String URL = urlBuilder.getUrlForWiki(Configuration.getWikiName()) + URLsContent.WIKI_DIR + articleTitle;
+    String URL = urlBuilder.getUrlForWiki() + URLsContent.WIKI_DIR + articleTitle;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename this local variable name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS00117)